codex-rs/core/tests/suite/compact.rs

```diff
@@ -366,7 +366,8 @@ async fn summarize_context_three_requests_and_instructions() {
     );
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+#[cfg_attr(windows, tokio::test(flavor = "multi_thread", worker_threads = 4))]
```

Comment:
Add a comment why we're doing this?

Note this is slightly concerning (and strange that it only happens on Windows) because it implies there is some sort of potential deadlock?

While it is nice for our tests to run faster, we should probably set `worker_threads=1` everywhere to tease out these types of issues.

I suspect we generally use values other than 1 because folks copy-pasta'd things...